apps/emqx_bridge/test/emqx_bridge_v2_api_SUITE.erl

```diff
@@ -1477,7 +1476,7 @@ t_cluster_later_join_metrics(Config) ->
             ?assertMatch(
                 {ok, 200, #{
                     <<"metrics">> := #{<<"success">> := _},
-                    <<"node_metrics">> := [#{<<"metrics">> := #{}}, #{<<"metrics">> := #{}} | _]
```

Comment:
Returns a single node result now, because BPAPI might not be ready / not have announced the new node's APIs yet, and thus is rejected when choosing the nodes.